report/data_tasks/report/create_from_scratch/04_lms/04_groups/03_organization_group/01_create_view.sql

```diff
@@ -3,26 +3,45 @@ DROP MATERIALIZED VIEW IF EXISTS lms.organization_group CASCADE;
 CREATE MATERIALIZED VIEW lms.organization_group AS (
     -- Use distinct here to simplify the model from LMS. We want one row per
     -- org, rather than each org / lms-group combo
-    SELECT DISTINCT group_id, organization_id
-    FROM (
-        SELECT
-        CONCAT('us-', group_id) AS group_id,
-        CASE
-            WHEN organization_id IS NULL THEN NULL
-            ELSE CONCAT('us-', organization_id)
-        END AS organization_id
-    FROM
-        lms_us.group_map
+    WITH organization_group AS (
+        SELECT DISTINCT group_id, organization_id
+        FROM (
+            SELECT
+            CONCAT('us-', group_id) AS group_id,
+            CASE
+                WHEN organization_id IS NULL THEN NULL
+                ELSE CONCAT('us-', organization_id)
+            END AS organization_id
+        FROM
+            lms_us.group_map
+
+        UNION ALL
 
-    UNION ALL
+        SELECT
+            CONCAT('ca-', group_id) AS group_id,
+            CASE
+                WHEN organization_id IS NULL THEN NULL
+                ELSE CONCAT('ca-', organization_id)
+            END AS organization_id
+        FROM
+            lms_ca.group_map
+        ) AS data
+    )
 
     SELECT
-        CONCAT('ca-', group_id) AS group_id,
-        CASE
-            WHEN organization_id IS NULL THEN NULL
-            ELSE CONCAT('ca-', organization_id)
-        END AS organization_id
-    FROM
-        lms_ca.group_map
-    ) AS data
+        -- We will include the group and organization names here even though
+        -- we could get it from the tables directly to avoid issues in Metabase.
+        -- It doesn't handle multiple columns with the same name (like `name`)
+        -- properly.
+        organizations.id AS organization_id,
+        organizations.name AS organization_name,
+        organizations.public_id AS organization_public_id,
```

Comment:
I've included the public id, because we nearly always want it in our reports, and it saves us a join. You can't present the name without it as a link, as the dashboard uses the public id as the key.